paths.go

```diff
@@ -114,12 +126,12 @@ func (p *Paths) initDataDirectory(dataDirectoryPath, appName string) (err error)
 }
 
 // AstilectronDownloadSrc returns the download URL of the (currently platform-independent) astilectron zip file
-func AstilectronDownloadSrc() string {
+func AstilectronDownloadSrc( VersionAstilectron string) string {
```

Comment:
Variable name should begin with a lower case letter : `versionAstilectron`

Reply:
Fixed